src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+
+  &__star {
+    display: flex;
+    background-image: url('/src/images/star.svg');
+    background-position: center;
+    height: 16px;
+    width: 16px;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &--4 {
+    .stars__star {
+      &:nth-child(-n + 4) {
+        background-image: url('/src/images/star-active.svg');
```

Comment:
As with the previous comment on line 6, ensure that the path to the image is reliable after SCSS compilation. Consider using a variable for the images path.